boto/vendored/regions/regions.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2014 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Unnecessary complication.

We literally need one class here.

I think there's too much emphasis placed on ease of backporting here.  After the initial PR was merged in feb last year, we've literally made a single line change to this module in all of 2016.  Sure we may need to backport things, but at the frequency of once a year, I don't think it's worth all this complication for such an infrequent task.  I'd rather go with the simpler approach.